db/seeds.rb

```diff
@@ -262,6 +262,9 @@ def seed_quantity(item_name, organization, storage_location, quantity)
   )
 end
 
+# Assign a value to all items so we can verify that totals are working
+Item.where(value_in_cents: 0).update_all(value_in_cents: 100)
```

Comment:
BEAUTIFUL. :100: :100: :100: 

I'm indifferent on the varied values, myself, but I respect @mdworken 's judgement on this, if he thinks we should include it.